include/flatbuffers/flatbuffers.h

```diff
@@ -343,6 +343,11 @@ template<typename T> static inline size_t VectorLength(const Vector<T> *v) {
 struct String : public Vector<char> {
   const char *c_str() const { return reinterpret_cast<const char *>(Data()); }
   std::string str() const { return std::string(c_str(), Length()); }
+#ifdef FLATBUFFERS_STRING_VIEW
```

Comment:
Please indent the #preprocessor commands like the rest of the file does.

Reply:
Done in 43394a6